mypy.ini

```diff
@@ -0,0 +1,15 @@
+[mypy]
+disallow_incomplete_defs = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_decorators = False
+disallow_untyped_defs = True
+follow_imports = 'normal'
+ignore_missing_imports = True
+no_implicit_optional = True
+strict_optional = True
+warn_no_return = True
+warn_redundant_casts = True
+warn_return_any = True
+warn_unused_configs = True
+warn_unused_ignores = True
```

Comment:
I recognise that this should have a newline but I would like to put fixing this off until a follow-up.